src/styles/blocks/card.scss

```
@@ -0,0 +1,96 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  color: $primary-text-color;
+
+  &__image {
+    margin: auto;
+    display: block;
+    height: 134px;
+    width: 160px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 1.5;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 1.4;
+    color: $secondary-text-color;
+    width: 100%;
+    margin-bottom: 16px;
+  }
+
+  &__rate {
+    display: flex;
+    width: 100%;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__counter {
+    font-size: 10px;
+    line-height: 1.4;
+    font-weight: 400;
+  }
+
+  &__price {
+    display: flex;
+    width: 100%;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__price-price {
+    font-size: 12px;
+    line-height: 1.5;
+    color: $secondary-text-color;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    line-height: 1.12;
+    font-weight: 700;
+  }
+
+  &__buy-button {
+    box-sizing: border-box;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 100%;
+    height: 40px;
+    background: $blue-accent;
+    border: 1px solid transparent;
+    border-radius: 5px;
+    color: #fff;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 1.14;
+    text-transform: uppercase;
+
+    transition: color 0.3s, background-color 0.3s;
+  }
+
+  &:hover .card__buy-button {
+    background-color: #fff;
+    color: $blue-accent;
+    border-color: $blue-accent;
+  }
```

You should use :hover only for button, not for card
```suggestion
    transition: color 0.3s, background-color 0.3s;
        
    &:hover {
      background-color: #fff;
      color: $blue-accent;
      border-color: $blue-accent;
    }
  }
```